### PR TITLE
Add a news post for SR2021 league 2 session

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

File: _posts/2021-02-06-sr2021-second-league-sessions.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: news
+title: SR2021 Teams Show Progress in Second League Session
+---
+
+With another 36 matches of the SR2021 Competition, team HRS3 from Hills Road
+Sixth Form College head up the league leaderboard, though a number of other
+teams are hot on their heels. This weekend's matches saw great perfomances from
+our teams, including three new teams which hadn't submitted a robot for the
+first league sesion.
```
Comment (RealOrangeOne) [inline _posts/2021-02-06-sr2021-second-league-sessions.md:10]:
```suggestion
first league session.
```

File: _posts/2021-02-06-sr2021-second-league-sessions.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+At the end of the league we announced to teams the details of our
+[third game module](https://studentrobotics.org/docs/resources/2021/rulebook.html).
+This changes the way that a robot's score is calculated from being the largest
+extent of their territory during the match to being purely based on the
+territory they control at the end of the match. Additionally the territories
+will now become locked (and unclaimed) after they have been successfully claimed
+four times.
+
+This module of the game requries that the teams robots further increase their
+awawreness of their environment and how they react to it if they want to claim
```
Comment (RealOrangeOne) [inline _posts/2021-02-06-sr2021-second-league-sessions.md:27]:
```suggestion
awareness of their environment and how they react to it if they want to claim
```

File: _posts/2021-02-06-sr2021-second-league-sessions.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* [League Session 1](https://www.youtube.com/watch?v=cAvk-nfTUis)
+
+We look forward to seeing you over the coming months for the continuation of
+the SR2021 Competition.
+
+| Date              | Competition Stage                                                                     |
+|-------------------|---------------------------------------------------------------------------------------|
+| 20th March 2021   | [League Session 3]({{ site.baseurl }}/events/sr2021/league-3/)                        |
+| 24th April 2021   | [League Session 4]({{ site.baseurl }}/events/sr2021/league-4/)                        |
+| 1st May 2021      | [Knockouts and Final]({{ site.baseurl }}/events/sr2020/virtual-competition-knockouts/)|
```
Comment (RealOrangeOne) [inline _posts/2021-02-06-sr2021-second-league-sessions.md:47]:
```suggestion
| 1st May 2021      | [Knockouts and Final]({{ site.baseurl }}/events/sr2021/knockouts/)|
```

Comment (PeterJCLaw) [reply _posts/2021-02-06-sr2021-second-league-sessions.md]: this was wrong in the previous post too it seems: 6ab2a70996b955be4b18b86eff018f9fef89d41a

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**